### PR TITLE
feat: optimize codes and reopen lint test ci

### PR DESCRIPTION
**Pull request by Xunzhuo**
Signed-off-by: Xunzhuo <mixdeers@gmail.com>

**What this PR does**:

Optimize codes and reopen CI

**Which issue(s) this PR fixes**:

Resolve https://github.com/mosn/layotto/issues/531

### CODE REVIEW

**Comment by Xunzhuo** [comment]
merge after https://github.com/mosn/layotto/pull/527

**Review by JervyShi — DISMISSED** [review]
LGTM

File: pkg/grpc/default_api/api.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ var (
 		},
 	}
 	// FIXME I put it here for compatibility.Don't write singleton like this !
-	// It should be refactored and deleted.
+	// LayottoAPISingleton It should be refactored and deleted.
```
Comment (seeflood) [inline pkg/grpc/default_api/api.go:62]: This sentence has two subjects.
I think it should be "LayottoAPISingleton should be refactored and deleted."

**Review by seeflood — CHANGES_REQUESTED** [review]

File: pkg/grpc/default_api/api.go (hunk 1 above)
Comment (Xunzhuo) [reply pkg/grpc/default_api/api.go]: Done

File: pkg/runtime/bindings/registry.go [hunk 2]
```diff
@@ -35,44 +35,44 @@ type Registry interface {
...
 
-type bindingsRegistry struct {
+type BindingsRegistry struct {
 	outputBindingStores map[string]func() bindings.OutputBinding
 	inputBindingStores  map[string]func() bindings.InputBinding
 	info                *info.RuntimeInfo
 }
 
-func NewRegistry(info *info.RuntimeInfo) *bindingsRegistry {
+func NewRegistry(info *info.RuntimeInfo) *BindingsRegistry {
```
Comment (seeflood) [inline pkg/runtime/bindings/registry.go:44]: There's no need to export `bindingsRegistry`. 
Instead, we can modify this constructor fuction to:
```go
func NewRegistry(info *info.RuntimeInfo) Registry {
```

**Review by seeflood — CHANGES_REQUESTED** [review]

File: pkg/runtime/bindings/registry.go (hunk 2 above)
Comment (Xunzhuo) [reply pkg/runtime/bindings/registry.go]: Good catch, this is a more elegant way!

File: pkg/runtime/bindings/registry.go [hunk 3]
```diff
@@ -35,44 +35,44 @@ type Registry interface {
 	CreateInputBinding(name string) (bindings.InputBinding, error)
 }
 
-type bindingsRegistry struct {
+type BindingsRegistry struct {
```
Comment (seeflood) [inline pkg/runtime/bindings/registry.go:38]: Coud u revert it to lowercase, `bindingsRegistry`

**Review by seeflood — CHANGES_REQUESTED** [review]

File: pkg/runtime/bindings/registry.go (hunk 3 above)
Comment (Xunzhuo) [reply pkg/runtime/bindings/registry.go]: Done @seeflood

**Review by seeflood — APPROVED** [review]
lgtm

**Review by zhenjunMa — APPROVED** [review]

**Pull request merged**